exonum/src/events/noise/wrapper.rs

```diff
@@ -126,9 +126,9 @@ impl NoiseWrapper {
 
         msg.chunks(NOISE_MAX_MESSAGE_LENGTH - TAG_LENGTH)
             .for_each(|msg| {
```

Comment:
Nit: `for` cycle seems more idiomatic Rust than `for_each`.

Reply:
Don't think so, it's just a matter of taste.

Reply:
Just looking [at Rust docs](https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.for_each):

> It's generally more idiomatic to use a `for` loop, but `for_each` may be more legible when processing items at the end of longer iterator chains.